ChangeLog

```diff
@@ -8,6 +8,7 @@ Tor Messenger 0.3.0b2 --
    * Permit storing cert. exceptions in private browsing mode
    * Bugzilla 1321420: Add a pref to disable JavaScript in browser requests
    * Bugzilla 1321641: Disable svg and mathml in content
+   * Trac 17517: Consider using different color for "Add Exception"
```

Comment:
This needs to be rebased and added to 0.3.0b3 ... but let me push a patch to start that first.